packages/server/lib/hooks/connection/post-connection.ts

```diff
@@ -123,7 +123,7 @@ async function execute(createdConnection: RecentlyCreatedConnection, providerNam
             }
         }
     } catch (err) {
-        await telemetry.log(LogTypes.POST_CONNECTION_SCRIPT_FAILURE, `Post connection manager failed, ${stringifyError(err)}`, LogActionEnum.AUTH, {
+        await telemetry.log(LogTypes.POST_CONNECTION_FAILURE, `Post connection manager failed, ${stringifyError(err)}`, LogActionEnum.AUTH, {
```

Comment:
I sort of lied when I said I didn't rename post_connection things (the one that are provider specific). I at least renamed the telemetry constants